auth-web/src/components/auth/BusinessContactForm.vue

```diff
@@ -0,0 +1,163 @@
+            type="tel"
+            v-mask="['(###)-###-####']"
+            v-model="phoneNumber"
+            :rules="phoneRules"
+          >
+          </v-text-field>
+        </v-flex>
+        <v-flex xs3>
+          <v-text-field
+            box label="Extension"
+            persistent-hint
+            :rules="extensionRules"
+            v-mask="'###'"
+            v-model="extension"
+          >
+          </v-text-field>
+        </v-flex>
+      </v-layout>
+    </div>
+    <div class="business-contact-form_row">
+      <v-layout wrap>
+        <v-spacer></v-spacer>
+        <v-btn class=".save-continue-button" @click="save" color="primary" large>
+          <span>Save and Continue</span>
+        </v-btn>
+        <v-btn class=".skip-button mr-0" @click="skip" color="secondary" large>
+          <span>Skip</span>
+        </v-btn>
+      </v-layout>
+    </div>
+    </v-form>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator'
+import { getModule } from 'vuex-module-decorators'
+import BusinessModule from '../../store/modules/business'
+import configHelper from '../../util/config-helper'
+import { mask } from 'vue-the-mask'
+
+@Component({
+  directives: {
+    mask
+  }
+})
+export default class BusinessContactForm extends Vue {
+  private VUE_APP_COPS_REDIRECT_URL = configHelper.getValue('VUE_APP_COPS_REDIRECT_URL')
+  private businessStore = getModule(BusinessModule)
+  private emailAddress = ''
+  private confirmedEmailAddress = ''
+  private phoneNumber = ''
+  private extension = ''
+  private formError = ''
+
+  private emailRules = [
+    v => !!v || 'Email address is required',
+    v => {
+      const pattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      return pattern.test(v) || 'Valid email is required'
+    }
+  ]
+
+  private phoneRules = [
+    v => (v.length === 0 || v.length === 14) || 'Phone number is invalid'
+  ]
+
+  private extensionRules = [
+    v => (v.length >= 0 || v.length <= 3) || 'Extension is invalid'
+  ]
+
+  private emailMustMatch (): string {
+    return (this.emailAddress === this.confirmedEmailAddress) ? '' : 'Emails addresses must match'
+  }
+
+  private isFormValid (): boolean {
+    return (this.$refs.form as Vue & { validate: () => boolean }).validate()
+  }
+
+  mounted () {
+    const contact = this.businessStore.currentBusiness.contact1
+    if (contact) {
+      this.emailAddress = this.confirmedEmailAddress = contact.emailAddress
+      this.phoneNumber = contact.phoneNumber
+      this.extension = contact.extension
+    }
+  }
+
+  save () {
+    if (this.isFormValid()) {
+      // TODO: Change this to transition to entity dashboard once complete
+      setTimeout(() => {
+        window.location.href = this.VUE_APP_COPS_REDIRECT_URL
+      }, 500)
+    }
+  }
+
+  skip () {
+    // Go directly to co-op UI without saving
```

Comment:
May be we need to set a session storage here, to handle the below scneario;
1) User logs in
2) clicks 'Skip'
3) Goes to co-ops, then accidentally by back button/or any action lands to login page.
4) I believe at this point user needs to be sent to Co-Ops again without asking the credentials and not showing the Business Contact Info UI - Since the user opted to not to fill the business contact info for that session ?

Reply:
After discussion with @pksumesh, will add a variable in session storage to indicate whether entering contact info has been skipped in that session or not.

Reply:
> After discussion with @pksumesh, will add a variable in session storage to indicate whether entering contact info has been skipped in that session or not.

or simply , we can always redirect to coops application if there is a token and if the page is not accessed directly.whichever is easier.